screens/ProfileScreen.js

```diff
@@ -1,55 +1,122 @@
 /* profile screen shows user data and has option to edit user data
 also has links to change pw, donate, faq*/
 
-/* replace later with actual display name component -> link to firebase */
-/* replace later with actual email component -> link to firebase */
-/* replace later with actual image component -> link to firebase */
+// TODO: BIANCA
+// fix image cancel bug -> update image url
+// pull email from firebase and display
```

Comment:
Take a look at utils/db/auth/auth.js for an example of how to pull name (pulling email should be very similar).